av/container/output.pyx

```diff
@@ -220,10 +224,9 @@ cdef class OutputContainer(Container):
 
         # Make another reference to the packet, as av_interleaved_write_frame
         # takes ownership of it.
-        cdef lib.AVPacket *packet_ptr = lib.av_packet_alloc()
-        self.err_check(lib.av_packet_ref(packet_ptr, packet.ptr))
+        self.err_check(lib.av_packet_ref(self.packet_ptr, packet.ptr))
 
         cdef int ret
         with nogil:
-            ret = lib.av_interleaved_write_frame(self.ptr, packet_ptr)
+            ret = lib.av_interleaved_write_frame(self.ptr, self.packet_ptr)
         self.err_check(ret)
```

Comment:
lib.av_interleaved_write_frame will reinitialize self.packet_ptr, so we can reuse it on subsequent calls: https://ffmpeg.org/doxygen/trunk/group__lavf__encoding.html#ga37352ed2c63493c38219d935e71db6c1